packages/next/test/unit/export.test.js

```diff
@@ -1,6 +0,0 @@
-describe('export', () => {
```

Comment:
This test was not very useful. And it started failing because `@vercel/next` was removed from the root level `package.json`.

Reply:
Hmm, I feel like this was added because of a regression but I suppose its not the valuable since no one runs the builders directly anymore so it should be implicitly tested by `vercel build` already 👍